modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -251,41 +251,36 @@ public <V> V runConsistently(WriteClosure<V> closure) throws StorageException {
         if (threadLocalWriteBatch.get() != null) {
             return closure.execute();
         } else {
-            if (!busyLock.enterBusy()) {
-                throw new StorageClosedException();
-            }
-
-            try (var writeBatch = new WriteBatchWithIndex()) {
-                threadLocalWriteBatch.set(writeBatch);
+            return busy(() -> {
```

Comment:
Aren't we concerned about allocations anymore? Each call to `runConsistently()` will allocate a `Supplier` index.

If we use it here, should we also use this pattern everywhere (at least, in all storages)?

Reply:
I think we won't care much about lambda allocation yet. 
For other MV storages I will do the same but in other tickets.